### PR TITLE
Update core-units.md

### PR DESCRIPTION
**Pull request by blimpa**
Sorry, @LongForWisdom. Had to reclone :man_shrugging: 

- Rewrote some of it for clarity and precision
- Added some links

### CODE REVIEW

**Review by LongForWisdom — APPROVED** [review]
Looks good by me, have Gala confirm :)

File: core-units/core-units.md [hunk 1]
```diff
@@ -3,26 +3,21 @@
...
 
-Each active Core Unit is made up of at least three active proposals:
-- An Accepted *mandate* proposal (MIP39).
-- One or more Accepted *budget* proposals (MIP40).
-- One or more Accepted *facilitator* proposals (MIP41).
+Each active Core Unit is made up of at least three active subproposals (i.e., subproposals that have not been superseded by other subproposals): a *mandate* subproposal (see [MIP39](https://mips.makerdao.com/mips/details/MIP39)) that lays out the objectives and directions that a given Core Unit is responsible for fulfilling. Executing these objectives usually requires funding, which is requested via *budget* subproposals (see [MIP40](https://mips.makerdao.com/mips/details/MIP40)). *Facilitator* subproposals (see [MIP41](https://mips.makerdao.com/mips/details/MIP41)) establish individuals as official points of contact and parties responsible for the Core Unit fulfilling its mandate.
```
Comment (blimpa) [inline core-units/core-units.md:8]: "is made up of" is kind of vague. "Formalized by", "instituted by means of"?

Comment (LongForWisdom) [reply core-units/core-units.md:8]: consists of?

Comment (Gala-G) [reply core-units/core-units.md:8]: Let's go with "Consists of"

**Review by Gala-G — APPROVED** [review]
It's perfect, Pablo. I approve all changes. Thank you

**Pull request merged**